src/oic/oic/provider.py

```diff
@@ -234,7 +234,8 @@ def __init__(self, name, sdb, cdb, authn_broker, userinfo, authz,
                  client_authn, symkey=None, urlmap=None, ca_certs="", keyjar=None,
                  hostname="", template_lookup=None, template=None,
                  verify_ssl=True, capabilities=None, schema=OpenIDSchema,
-                 jwks_uri='', jwks_name='', baseurl=None, client_cert=None):
+                 jwks_uri='', jwks_name='', baseurl=None, client_cert=None,
+                 form_post_template=FORM_POST):
```

Comment:
A name that hints it is about the form_post response_mode would be better.

The provider already has two template related args (template_lookup/template). Why not reuse the template_lookup to lookup a custom form_post response template too?

Reply:
Good catch. I can probably reuse the `template` arg.

I am not sure how (and for what) is the `template_lookup` used as its default value would cause error.